packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1104,9 +1112,15 @@ export default class BackupNg {
                         await chainVhd(handler, parentPath, handler, path)
                       }
 
-                      return handler.getSize(path)
+                      return {
+                        isDelta,
+                        size: await handler.getSize(path),
+                      }
                     })
-                  ).then(sum)
+                  ).then(result => ({
+                    size: sumBy(result, 'size'),
+                    isFull: some(result, { isDelta: false }),
+                  }))
```

Comment:
This should be implemented for CR as well.

IMHO, you should do it before the `waitAll`.